website/docs/api/architectures.md

```diff
@@ -631,11 +632,22 @@ model simply adds a linear layer with softmax activation to predict scores given
 the token vectors.
 
 | Name        | Description                                                                                |
-| ----------- | ------------------------------------------------------------------------------------------ |
+|-------------|--------------------------------------------------------------------------------------------|
 | `tok2vec`   | Subnetwork to map tokens into vector representations. ~~Model[List[Doc], List[Floats2d]]~~ |
 | `nO`        | The number of tags to output. Inferred from the data if `None`. ~~Optional[int]~~          |
+| `normalize` | Normalize probabilities during inference. Defaults to `False`. ~~bool~~           |
```

Comment:
Nitpick: could you run `prettier` on the markdown formatting? :-)

Reply:
Done!